packages/react-resize-handle/src/hooks/useMouseHandler.ts

```diff
@@ -38,7 +35,7 @@ export function useMouseHandler(params: UseMouseHandlerParams) {
         clientY - dragStartOriginCoords.current.clientY,
       ];
 
-      let newValue = initialElementSize.current;
+      let newValue = startValue.current || 0;
```

Comment:
```suggestion
      let newValue = startValue.current ?? 0;
```

Reply:
actually is the conditional assignment even necessary? `startValue.current` is always a number

Reply:
yea, not anymore, removing